analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/HashesShouldHaveUnpredictableSaltTest.cs

```diff
@@ -76,13 +67,18 @@ public class HashesShouldHaveUnpredictableSaltTest
 
 #if NET
 
+    [TestMethod]
+    public void HashesShouldHaveUnpredictableSalt_Roslyn_CSharp8() =>
+        roslynCS.AddPaths("HashesShouldHaveUnpredictableSalt.CSharp8.cs")
+            .WithOptions(ParseOptionsHelper.FromCSharp8)
+            .Verify();
+
```

Comment:
Why has this moved to `#if NET`?

Reply:
These features are not supported by the .NET Framework (e.g. interfaces having method implementations).